project-base/app/src/FrontendApi/Mutation/Payment/PaymentMutation.php

```diff
@@ -47,17 +47,20 @@ public function payOrderMutation(Argument $argument): PaymentSetupCreationData
 
     /**
      * @param \Overblog\GraphQLBundle\Definition\Argument $argument
-     * @return bool
+     * @return array{isPaid: bool, transactionCount: int}
      */
-    public function checkPaymentStatusMutation(Argument $argument): bool
+    public function checkPaymentStatusMutation(Argument $argument): array
```

Comment:
I feel like check method should return bool, so can you change name to get?

Reply:
If we change it to get, it should not be a mutation then... Or maybe it should, as it probably checks with the external service?

Reply:
I agree with the Seba. It's a mutation, so I would not name it with the get prefix

Reply:
In that case `updatePaymentStatusMutation`?

Reply:
renamed to updatePaymentStatusMutation along with the usages